staging/src/k8s.io/kubectl/pkg/cmd/version/version.go

```diff
@@ -124,6 +126,15 @@ func (o *Options) Run() error {
 		// Always request fresh data from the server
 		o.discoveryClient.Invalidate()
 		serverVersion, serverErr = o.discoveryClient.ServerVersion()
+		// FIXME soltysh:
+		// since we have defaulting to localhost:8080 in staging/src/k8s.io/client-go/tools/clientcmd/client_config.go#getDefaultServer
+		// we need to hack around this when getting version and ignore reaching the localhost:8080 server
+		switch t := serverErr.(type) {
+		case *url.Error:
+			if strings.Contains(t.URL, "localhost:8080") {
```

Comment:
why is this needed? 1.17 would print an error if you did `kubectl version` with no config file

Reply:
Dropped.